src/theory/theory_model_builder.cpp

```diff
@@ -21,6 +21,7 @@
 #include "options/smt_options.h"
```

Comment:
Can we remove this?

Reply:
Its still required for options::debugCheckModels() unfortunately.